assets/src/edit-story/components/library/panes/media/common/InnerElement.js

```diff
@@ -0,0 +1,184 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import styled, { css } from 'styled-components';
+import { rgba } from 'polished';
+import { useCallback, useMemo } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import { getSmallestUrlForWidth } from '../../../../../elements/media/util';
+import resourceList from '../../../../../utils/resourceList';
+import { useDropTargets } from '../../../../dropTargets';
+
+const styledTiles = css`
+  width: 100%;
+  cursor: pointer;
+  transition: 0.2s transform, 0.15s opacity;
+  border-radius: 4px;
+  opacity: 0;
+`;
+
+const Image = styled.img`
+  ${styledTiles}
+`;
+
+// Display the newly uploaded videos without a delay: showWithoutDelay
+const Video = styled.video`
+  ${styledTiles}
+  object-fit: cover;
+  ${({ showWithoutDelay }) => (showWithoutDelay ? 'opacity: 1;' : '')}
+`;
+
+const Duration = styled.div`
+  position: absolute;
+  bottom: 8px;
+  left: 8px;
+  background: ${({ theme }) => rgba(theme.colors.bg.workspace, 0.6)};
+  font-family: ${({ theme }) => theme.fonts.duration.family};
+  font-size: ${({ theme }) => theme.fonts.duration.size};
+  line-height: ${({ theme }) => theme.fonts.duration.lineHeight};
+  letter-spacing: ${({ theme }) => theme.fonts.duration.letterSpacing};
+  padding: 0 6px;
+  border-radius: 10px;
+`;
+
+const HiddenPosterImage = styled.img`
+  display: none;
+`;
+
+function InnerElement({
```

Comment:
It seems this file is named `InnerElement.js` with a capital `I`. It should be a lowercase first letter in filenames.